ios/Shared/RocketChat/Database.swift

```diff
@@ -25,7 +25,9 @@ final class Database {
   init(server: String) {
     if let url = URL(string: server) {
       if let domain = url.domain, let directory = directory {
-        self.database = WatermelonDB.Database(path: "\(directory)/\(domain)-experimental.db")
+        if let isOfficial = Bundle.main.object(forInfoDictionaryKey: "IS_OFFICIAL") as? Bool {
```

Comment:
Notification related features not will work on white label apps that doesn't have `IS_OFFICIAL` key using this implementation.
Let's make it something like
```swift
let isOfficial = Bundle.main.object(forInfoDictionaryKey: "IS_OFFICIAL") as? Bool ?? false
self.database = WatermelonDB.Database(path: "\(directory)/\(domain)\(isOfficial ? "" : "-experimental").db")
```